.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v2
+        with:
+          java-version: 11
+      - name: Run tests
+        run: csbt +compile +test scalafmtCheckAll scalastyle
```

Comment:
Is there a reason we're using `csbt` instead of `sbt`? `setup-scala` says "This launcher does not work with all builds, only use csbt if you know what you are doing."

Reply:
It works for our build and has faster startup time according to the README.

> For faster startup, the csbt command is installed using the Coursier-based coursier/sbt-extras launcher.